components/ui/MenuBar.js

```diff
@@ -81,8 +83,8 @@ const Toolbar = ({ brand, children }) => {
         <li className="brand">
           <Link href="/" passHref>
             <>
-              {brand}
-              <ChevronIcon />
+              <LevelLogoIcon />
+              Level Protocol
```

Comment:
@edderrd — Just wondering why we have the date values as links? Can we just have those as static content? (I’d taken the previous `<a>`s and converted them to next/link without thinking about this so I own it too, but I doubt we’ll make them into anything at present.)

Reply:
Note: this comment is above the change; it’s as close to the items as possible.

Reply:
converted to tray items to avoid using those links

Reply:
That’s a good solution @edderrd — will you remove the hover on `tray-item` since it’s not an actionable item.

Reply:
done and also remove active to avoid confusion when clicked

Reply:
Thanks!